client/views/home.ejs

```diff
@@ -140,6 +140,14 @@
       $('.preloader').fadeOut(1500);
     });
   </script>
+<!--  this is for login toast -->
+  <script>
+    $(window).on('load', function() {
+      setTimeout(function() {<% if (!locals.user) { %>
```

Comment:
I'd move the script to an own file and just reference it from there if you need to call the script twice